app/uk/gov/hmrc/nisp/models/StatePension.scala

```diff
@@ -79,7 +79,8 @@ case class StatePension(earningsIncludedUpTo: LocalDate,
                         numberOfQualifyingYears: Int,
                         pensionSharingOrder: Boolean,
                         currentFullWeeklyPensionAmount: BigDecimal,
-                        reducedRateElection: Boolean) {
+                        reducedRateElection: Boolean,
+                        abroadAutoCredits: Boolean) {
```

Comment:
I would consider writing some custom `Reads` logic here to allow abroadAutoCredits to be optional.
This should have been done for reduced rate election, it allows the frontend to support both versions of the API. It means you don't have to revert changes if something goes wrong, you can just roll back and it will make the deployment seemless if you deploy the frontend first.

Example.
https://github.com/hmrc/state-pension/blob/master/app/uk/gov/hmrc/statepension/domain/nps/NpsSummary.scala#L46